.circleci/config.yml

```diff
@@ -300,6 +301,33 @@ commands:
             circleci tests glob "tests/**/*.spec.ts" | circleci tests split --split-by=timings > /tmp/tests-to-run
             yarn test:ci $(cat /tmp/tests-to-run)
           no_output_timeout: 15m
+      - run:
+          name: Create circleci_parameters.json
+          working_directory: ~/workbench/e2e
+          command: |
+            echo "{ \
+              \"circleBranch\": \"${CIRCLE_BRANCH}\", \
+              \"circleJob\": \"${CIRCLE_JOB}\", \
+              \"circleBuild\": \"${CIRCLE_BUILD_NUM}\", \
+              \"circleBuildUrl\": \"${CIRCLE_BUILD_URL}\", \
+              \"circleNodeIndex\": \"${CIRCLE_NODE_INDEX}\" \
+              }" \
+              > ./logs/circleci_parameters.json
+          when: always
+      - run:
+          name: Add CircleCI job data to test results json before upload json to storage bucket
+          working_directory: ~/workbench/e2e/logs
+          command: |
+            NEW_JSON=circleci-build-${CIRCLE_BUILD_NUM}-${CIRCLE_NODE_INDEX}-test-results.json
+            jq -s '.[0] * .[1]' test-results.json circleci_parameters.json > ${NEW_JSON}
+            # Remove all whitespaces
+            cat ${NEW_JSON} | tr '\n' ' ' > tmp.json && mv tmp.json ${NEW_JSON}
+            # Upload json to storage bucket
+            PROJECT_NAME=all-of-us-workbench-test
+            STORAGE_BUCKET=gs://circleci-puppeteer-test-results
+            gcloud config set project ${PROJECT_NAME}
```

Comment:
This has implicit side-effects on other gcloud commands, in general I recommend avoiding setting this in most cases. This line also should not be necessary at all for running the following gsutil command. I would remove it and the project name.

If you still have problems after this, please let me know, but you can instead pass an explicit project ID flag to gsutil.

Reply:
👍 
Addressing latest feedback: Use jq `-c`, and `gsuite cp` works after removing set project. Thank you CH.